<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>797</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>770</width>
        <height>555</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>458</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>376</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(255,255,255,255);
border-bottom-left-radius: 50px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignLeft">
           <widget class="QPushButton" name="CloseBtn">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">#CloseBtn{
	image: url(:/icon/x.svg);
}
#CloseBtn:hover{
	background-color: rgb(222, 222, 222);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../Resource/resource.qrc">
              <normaloff>:/icon/x_bl.svg</normaloff>:/icon/x_bl.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item alignment="Qt::AlignHCenter">
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <family>Montserrat Medium</family>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>SIGN UP</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>7</number>
                </property>
                <item>
                 <widget class="QFrame" name="frame_9">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLineEdit" name="username">
                     <property name="minimumSize">
                      <size>
                       <width>301</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>9</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: 2px solid rgba(0, 0, 0, 0);
border-bottom-color: rgba(46, 82, 101, 200);
color: rgb(0, 0, 0);
padding-bottom: 7px;
</string>
                     </property>
                     <property name="placeholderText">
                      <string>User Name</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_7">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item alignment="Qt::AlignLeft">
                    <widget class="QLineEdit" name="email">
                     <property name="minimumSize">
                      <size>
                       <width>301</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>9</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: 2px solid rgba(0, 0, 0, 0);
border-bottom-color: rgba(46, 82, 101, 200);
color: rgb(0, 0, 0);
padding-bottom: 7px;
</string>
                     </property>
                     <property name="echoMode">
                      <enum>QLineEdit::Normal</enum>
                     </property>
                     <property name="placeholderText">
                      <string>Email</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_11">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QLineEdit" name="password">
                     <property name="minimumSize">
                      <size>
                       <width>301</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>9</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: 2px solid rgba(0, 0, 0, 0);
border-bottom-color: rgba(46, 82, 101, 200);
color: rgb(0, 0, 0);
padding-bottom: 7px;
</string>
                     </property>
                     <property name="echoMode">
                      <enum>QLineEdit::Password</enum>
                     </property>
                     <property name="placeholderText">
                      <string>Password</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="toggle_password_button">
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_10">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLineEdit" name="confirmpassword">
                     <property name="minimumSize">
                      <size>
                       <width>301</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>9</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgba(0, 0, 0, 0);
border: 2px solid rgba(0, 0, 0, 0);
border-bottom-color: rgba(46, 82, 101, 200);
color: rgb(0, 0, 0);
padding-bottom: 7px;
</string>
                     </property>
                     <property name="echoMode">
                      <enum>QLineEdit::Password</enum>
                     </property>
                     <property name="placeholderText">
                      <string>Confirm Password</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="toggle_password_button_2">
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_check">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="check_signup">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(255, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_5">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Maximum</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>2</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="signupbutton">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>34</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>8</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#signupbutton {
	background-color: rgba(65, 90, 112, 255);
	color: rgba(255, 255, 255, 200);
	border-radius: 5px;
}

#signupbutton:pressed {
	padding-left: 5px;
	padding-top: 5px;
	background-color: rgba(255, 107, 107, 255);
}

#signupbutton:hover {
	background-color: rgba(255, 107, 107, 255);
}</string>
                  </property>
                  <property name="text">
                   <string>SIGN UP</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Maximum</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>3</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft|Qt::AlignTop">
              <widget class="QFrame" name="frame_8">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Maximum</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>5</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <pointsize>9</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Have an account?</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignLeft">
                 <widget class="QPushButton" name="gotologinbutton">
                  <property name="font">
                   <font>
                    <pointsize>9</pointsize>
                    <underline>true</underline>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background: transparent;
color:rgb(0, 85, 255);
text-decoration: underline;</string>
                  </property>
                  <property name="text">
                   <string>Login now</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_12">
         <property name="minimumSize">
          <size>
           <width>373</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>373</width>
           <height>539</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/image/bg-login.png);
border-top-right-radius: 50px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
